torch/_torch_docs.py

```diff
@@ -3299,9 +3306,17 @@ def merge_dicts(*dicts):
 
     >>> torch.fmod(torch.tensor([-3., -2, -1, 1, 2, 3]), 2)
     tensor([-1., -0., -1.,  1.,  0.,  1.])
-    >>> torch.fmod(torch.tensor([1, 2, 3, 4, 5]), 1.5)
+    >>> torch.fmod(torch.tensor([1, 2, 3, 4, 5]), -1.5)
     tensor([1.0000, 0.5000, 0.0000, 1.0000, 0.5000])
 
+.. seealso::
+
+        :func:`torch.fmod`, which computes the element-wise remainder of
+        division equivalently to the C library function ``fmod()``.
```

Comment:
This seems redundant as this points back to the same documentation. (Maybe add this line as a note?)

Reply:
Oh my bad! This shouldn't be there in `torch.fmod`. Maybe I can link `torch.remainder` here instead?